### ISSUE TITLE
presence of Amelanchier intermedia in Nfld.?

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#2577]
Re: http://data.canadensys.net/vascan/taxon/8626?lang=en

Campbell et al. in FNA, list Amelanchier intermedia from insular NL, yet its not included in VASCAN from the island.  Did their NL specimens prove to be misIDs?

**Comment by brouille** [comment]
No. I merely was not aware they had added the taxon to Nfld. Reviewing the specimens, I found out that several old (Ayre, Spach, Knowling) had reported it from the Avalon.  I found back specimens in the MT db (no specimens at MT, however).

It is likely that Campbell saw some specimens since duplicates are at GH. Otherwise he would have merely mentioned it. In 2000, it had been excluded due to misidentification. I saw no discussion by Campbell. 

I am adding it to Nfld. in VASCAN. If it is erroneous, it can be excluded again, I guess.

Luc
